### ISSUE TITLE
As an EVCM, I want to read the site content in Spanish.

### ISSUE THREAD

**Issue by switzersc-usds** [usds/justice40-tool#184]
Extracting this from #16 because it is blocked by #99. 

**Description**
We need to set up language switching on our site so users can switch between English and Spanish content. 

**Solution**
Once content for About, Explore the tool, Data & methodology, Contact is finalized and approved, get Spanish translations and make available to visitors to the site

**Tasks**
- [ ] Jane confirm design for accessing Spanish version of pages (I believe this is already done)
- [ ] Get translations from Department of State once all site content in English is finalized and gone through approval
- [ ] Team member fluent Spanish reader/speaker review what DoS provides
- [ ] Dev Incorporate Spanish content onto site
- [ ] Implement language switching

**Definition of Done**
- [ ] User can read all pages on site in English
- [ ] User can read all pages on site in Spanish

**Comment by katherinedm-usds** [comment]
To help us be hyper focused on July 27th in sprint planning, any cards that were in product backlog that are no longer MVP for July 27th we're sending back to Icebox. Because it will be challenging enough just to get English content reviewed and approved in time, we're taking Spanish for July 27th off the table. We do know however that translation is very doable via DoS and can maybe expect a ~1.5 week turnaround time for that.
